lib/generators/avo/templates/locales/avo.uk.yml

```
@@ -0,0 +1,120 @@
+---
+uk:
+  avo:
+    resource_translations:
+      product:
+        save: "Зберегти продукт!"
+    action_ran_successfully: Дія успішно виконана!
+    actions: Дії
+    and_x_other_resources: і ще %{count} інших ресурсів
+    applied: застосовано
+    are_you_sure: Ви впевнені?
+    are_you_sure_detach_item: Ви впевнені, що хочете відокремити цей %{item}?
+    are_you_sure_you_want_to_run_this_option: Ви впевнені, що хочете виконати цю дію?
+    attach: Прикріпити
+    attach_and_attach_another: Прикріпити і ще один
+    attach_item: Прикріпити %{item}
+    attachment_class_attached: "%{attachment_class} прикріплено."
+    attachment_class_detached: "%{attachment_class} відокремлено."
+    attachment_destroyed: Вкладення знищено
+    cancel: Скасувати
+    choose_a_country: Виберіть країну
+    choose_an_option: Виберіть опцію
+    choose_item: Виберіть %{item}
+    clear_value: Очистити значення
+    click_to_reveal_filters: Натисніть, щоб показати фільтри
+    close_modal: Закрити модальне вікно
+    confirm: Підтвердити
+    create_new_item: Створити новий %{item}
+    dashboard: Панель керування
+    dashboards: Панелі керування
+    default_scope: Всі
+    delete: Видалити
+    delete_file: Видалити файл
+    delete_item: Видалити %{item}
+    detach_item: Відокремити %{item}
+    details: Деталі
+    download: Завантажити
+    download_file: Завантажити файл
+    download_item: Завантажити %{item}
+    edit: Редагувати
+    edit_item: Редагувати %{item}
+    empty_dashboard_message: Додайте картки до цієї панелі керування
+    failed: Не вдалося
+    failed_to_find_attachment: Не вдалося знайти вкладення
+    failed_to_load: Не вдалося завантажити
+    file:
+      one: файл
+      other: файлів
+    filter_by: Фільтрувати за
+    filters: Фільтри
+    go_back: Назад
+    grid_view: Сітковий вигляд
```

```suggestion
    grid_view: Плитка
```


that's the word used here https://rozetka.com.ua/ua/routers/c80193/